src/main/webapp/css/style.css

```
@@ -390,48 +392,12 @@ simple mistaak .img-extrasmall {
 .img-large {
   width: 100px;
 }
+/*# sourceMappingURL=style.css.map */
 
-#logoup{
-
-  padding-top: 2vh;
-  padding-left: 2vh;
-}
-
-.cs{
-  box-shadow: -8px 4px 15px 0 grey;
-  position: absolute;
-  left: 50%;
-  top: 50%;
-  transform: translate(-50%, -50%);
-  padding: 10px;
-  background-color: lightgray;
-  text-align: center;
-  margin: 2vh;
-  padding: 5vh;
-}
-
-
-#logosq{
-  height: 8vh;
-  width: 8vw;
+#content-userprofile{
```

You need a space here (I know the old version didn't follow that but I did leave the comment in that PR too). 
```suggestion
#content-userprofile {
```

---

I'll fix it